src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditorViewModel.java

```diff
@@ -174,6 +174,17 @@ public void addNewFile() {
         }
     }
 
+    public void addNewManualFile(LinkedFile linkedFile) {
+
```

Comment:
Newlines at the start of a block aren't permitted as per our style convention. Try to check the details of the failing test below.
Then, follow the instructions regarding how to conform to checkstyle that you were told about in the last closed PR (import order, declaration order, etc.).

Reply:
I have removed the blank line at the start of the block and committed those changes. Could you please provide more details about the failing test you mentioned? Thanks

Reply:
@subhramit I have removed the extra blank lines and addressed the Checkstyle errors. However, I'm unsure whether any of the tests are failing due to the changes or if they were already in this state.

Could you please verify if the test failures are related to the recent changes?

Reply:
The tests are passing now, so no need to worry.